sass/tna-toolkit/tna-toolkit.scss

```diff
@@ -9,6 +9,7 @@
 html {
   box-sizing: border-box;
   -ms-overflow-style: scrollbar;
+  scroll-behavior: smooth;
```

Comment:
Note: This allows scrolling two jump-links (and other anchors) in CSS3, I've applied this change site-wide. Is there any reason we should not do this?

Reply:
I think we can use this site wide